src/Test/Proxy/NginxProxy.php

```diff
@@ -15,7 +15,7 @@ class NginxProxy extends Abstractproxy
 {
     protected $binary = 'nginx';
     protected $configFile;
-    protected $port = 8080;
+    protected $port = 8088;
```

Comment:
maybe our phpunit.xml changes the value through a constant? when running nginx and varnish on the same system, they will need to run on separate ports maybe.